shiny/_main.py

```diff
@@ -513,18 +513,38 @@ def try_import_module(module: str) -> Optional[types.ModuleType]:
     "-g",
     help="The GitHub URL of the template sub-directory. For example https://github.com/posit-dev/py-shiny-templates/tree/main/dashboard",
 )
+@click.option(
+    "--directory",
```

Comment:
Up to you... but py-shinylive has `--dir` for output directory values

We'd only have to update the `create()` method

Reply:
Oh yeah may as well keep them consistent